cmd/embedded-cluster/network_test.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func Test_getCIDRs(t *testing.T) {
```

Comment:
don't see a use case for when only `cidr` flag is used, but won't block pr on it

Reply:
Done via -https://github.com/replicatedhq/embedded-cluster/pull/1355/commits/6febab9af8c35aa78351c089f465efec0ab434d4